content/en/tracing/trace_collection/library_config/php.md

```diff
@@ -120,8 +120,8 @@ Whether to enable the endpoint data collection in profiles. Added in version `0.
 `DD_PROFILING_ALLOCATION_ENABLED`
 : **INI**: `datadog.profiling.allocation_enabled`. INI available since `0.88.0`.<br>
 **Default**: `1`<br>
-Enable the allocation size and allocation bytes profile type. Added in version `0.88.0`.<br>
-**Note**: This supersedes the `DD_PROFILING_EXPERIMENTAL_ALLOCATION_ENABLED` environment variable (`datadog.profiling.experimental_allocation_enabled` INI setting), which was available since `0.84`. If both are set, this one takes precedence.
+Enable the allocation size and allocation bytes profile type. Added in version `0.88.0`. When an active JIT is detected, allocation profiling will be turned off due to a bug in the ZendEngine.<br>
```

Comment:
```suggestion
Enable the allocation size and allocation bytes profile type. Added in version `0.88.0`. When an active JIT is detected, allocation profiling is turned off due to a limitation of the ZendEngine.<br>
```

Reply:
We don't typically list bugs in the documentation -- is this something that is expected to be resolved/changed in the near future? If so, we can create an additional card to track that, and update this section again when it's resolved.
If this is not expected to change in the near term, do you think the language suggested above would be sufficient?

Reply:
It should be fixed in the upcoming PHP versions 8.2.8 and 8.1.21 as soon as they are released. But we need to double check this when the release has been tagged